client/src/components/FieldHolder/FieldHolder.js

```diff
@@ -123,26 +143,34 @@ function fieldHolder(Field) {
   }
 
   FieldHolder.propTypes = {
-    leftTitle: React.PropTypes.any,
-    rightTitle: React.PropTypes.any,
-    title: React.PropTypes.any,
-    extraClass: React.PropTypes.string,
-    holderId: React.PropTypes.string,
-    id: React.PropTypes.string,
-    description: React.PropTypes.any,
-    hideLabels: React.PropTypes.bool,
-    message: React.PropTypes.shape({
-      extraClass: React.PropTypes.string,
-      value: React.PropTypes.any,
-      type: React.PropTypes.string,
+    leftTitle: PropTypes.any,
+    rightTitle: PropTypes.any,
+    title: PropTypes.any,
+    extraClass: PropTypes.string,
+    holderId: PropTypes.string,
+    id: PropTypes.string,
+    description: PropTypes.any,
+    hideLabels: PropTypes.bool,
+    message: PropTypes.shape({
+      extraClass: PropTypes.string,
+      value: PropTypes.any,
+      type: PropTypes.string,
     }),
+    data: PropTypes.oneOfType([
+      PropTypes.array,
```

Comment:
Should data be object not array It could be an object without prefix / suffix right?